h/services/search_index/service.py

```diff
@@ -76,7 +76,7 @@ def add_annotations_between_times(self, start_time, end_time, tag):
         :type start_time: datetime.datetime
         :type end_time: datetime.datetime
         """
-        self._queue.add_annotations_between_times(start_time, end_time, tag)
+        indexer.add_annotations_between_times.delay(start_time, end_time, tag)
```

Comment:
This is the actual change here: call the Celery task (which calls the queue) instead of calling the queue directly

Reply:
The more I look at this stuff, the more I think we should just put the tasks with the search service. I was thinking "what if you want to know what all the tasks are?" It seems to me two decent ways of going about that are "grep for celery.task" (our code is pretty grep dependant anyway), or look at the celery config where it lists exactly where all the modules come from.

Reply:
I think I'd rather keep Celery out of the services. I think I like keeping Celery in `h/tasks/` and having the Celery tasks mostly just be one-liners that call services. Rather than distributing Celery throughout the codebase